modules/engage-theodul-plugin-tab-shortcuts/src/main/resources/static/main.js

```diff
@@ -189,13 +189,14 @@ define(["jquery", "underscore", "backbone", "engage/core"], function($, _, Backb
             prepareCategories();
             var scuts = Engage.model.get("meInfo").get("shortcuts");
             if (scuts) {
-                $.each(scuts, function(i, v) {
-                    var cat = v.name.split(".")[0]
-                    if (cat !== undefined && shortcuts[cat] !== undefined) {
+                $.each(scuts, function(cat, val) {
+                    if (cat !== undefined && shortcuts[cat] != undefined && val != undefined) {
```

Comment:
2 or 4 spaces for indentation? Should be at least be consistent in the same file.

Reply:
The whole thing is an indentation mess anyway. I'll plan to apply the common ESLint rules used elsewhere to these modules anyway (alike pull request #923) which will then fix that. But I wanted to wait for this pull request. So, I guess we can ignore it here.

That said, if you want to do that @mliradelc, please go ahead 😉

Reply:
hi @lkiesow, I installed ESLint in VS code with the standard ruleset but I don't know if it's good for us, because, the common ESLint enforces to:

- Not use a semicolon and the end of a variable declaration
- Change all the variables to the Camel case form.

The first is very dangerous because can create unintended behavior in the code.

Ex:

```javascript
const hey = 'hey'
const you = 'hey'
const heyYou = hey + ' ' + you

['h', 'e', 'y'].forEach((letter) => console.log(letter))
```

Can  be interpreted like this:
```javascript
const hey = 'hey';
const you = 'hey';
const heyYou = hey + ' ' + you['h', 'e', 'y'].forEach((letter) => console.log(letter))
```

Generating an error `Uncaught TypeError: Cannot read property 'forEach' of undefined` Source: [Semicolons in Javascript](https://flaviocopes.com/javascript-automatic-semicolon-insertion/)
 
If we are trying to apply an ESLint standard, my vote goes to the AirBnB standard.